packages/extension/src/background.ts

```diff
@@ -12,6 +12,12 @@ setPlatform(new ExtensionPlatform());
 class ExtensionBackground {
     app = new App(new AjaxSender(process.env.PL_SERVER_URL!));
 
+    _autoLockAlarmName = "pl_autoLock";
+
+    get _lockDelayInMinutes() {
```

Comment:
A lot of code here is _similar_ (but not the same) as the `AutoLock` class, so I initially tried to make that work. In the end I was still duplicating most code, and since these alarms are very specific to the `background` piece of the `extension`, I ended up leaving them here.